apps/google-analytics-4/frontend/src/components/main-app/ChartContent.tsx

```diff
@@ -0,0 +1,31 @@
+import { Row, RunReportResponse } from '@/types';
+import LineChart from './line-chart/LineChart';
+
+interface Props {
+    pageViewData: RunReportResponse;
+}
+
+const ChartContent = (props: Props) => {
```

Comment:
Really like how you made this component abstraction to support LineChart as a reusable component 👍 !